Tasks/UsePythonVersion/taskutil.ts

```diff
@@ -0,0 +1,21 @@
+import * as task from 'vsts-task-lib/task';
+
+// TODO move this to vsts-task-lib 
+export enum Platform {
+    Windows,
+    MacOS,
+    Linux
+}
+
+/**
+ * Determine the operating system the build agent is running on.
+ * TODO move this to vsts-task-lib
+ */
+export function getPlatform(): Platform {
```

Comment:
Most other tasks that have to check platform do a simple string comparison. If you do end up adding this to the task lib, we should also add a user story to clean up all the other tasks.

Reply:
That would be nice.  The current way as I recall is to check `taskLib.osType().match(/^Win/)` something.  But that only tells you whether you're on Windows or not Windows, and is a bit more complicated than necessary.